### ISSUE TITLE
Setup LibreCores blog

### ISSUE THREAD

**Issue by imphil** [librecores/librecores-web#218]
For our upcoming newsletter (at least) we need a blog on LibreCores. We already have a disabled wordpress installation in our tree, but I'm not convinced that's the best way forward.

Options:
1. Use wordpress and theme it to match LibreCores
1. Integrate a Symfony-based blog solution like sonata news.
1. Go for a static site generator with blog support, e.g. Jekyll.

Maintaining the wordpress installation is a bit of a pain as it has a huge amount of functionality and hence attack vectors. So I'd like to avoid that if possible.

I then tried to setup Sonata News, but that requires setting up a huge amount of dependencies inside our Symfony setup (essentially all other sonata bundles), that doesn't look like a nice solution either.

So I'm thinking about just using Jekyll, and we already have a (not really maintained or used) setup for that at help.librecores.org. 

That gives a couple open questions:
- Use the existing approach of having Jenkins build the site and then push it somewhere, or integrate it into the standard librecores server setup?
- Domain name/URL: blog.librecores.org sounds like a plan.
- help.librecores.org sounds like a reasonable domain name for the remainder of our static content, which is currently part of the main site. This includes essentially all pages below https://www.librecores.org/static/, e.g. https://www.librecores.org/static/docs or https://www.librecores.org/static/community-hub.

I'll give this a bit more thought over the weekend, @wallento or @agathver if you have any additional points to make please let me know.

**Comment by agathver** [comment]
Use medium? It has a wider audience.

**Comment by olofk** [comment]
I'm not directly involved with the infrastructure, but as a potential future blog author, I'd say a static blog generator from markdown or similar works fine for me

**Comment by imphil** [comment]
I took for now the easiest (and for me most interesting, since new) route of creating a "publication" on Medium (as blogs are called there). So we have https://medium.com/librecores for now with basic "styling". I decided to go for a "LibreCores" account instead of an El Correo Libre one, as we'd like to get content wider than just the monthly newsletter out there (and stories can appear only in one publication on Medium). I think people with a Medium account can now submit stories to this publication, we'll see how it goes once we publish out first newsletter next week.

Remaining work:
- Add the RSS feed to our blog planet
- Link the page everywhere
- Make it public it exists (makes more sense once the initial content is out there, so waiting with that until next week.)
